### ISSUE TITLE
Creating a Training and Predicting Pipeline

### ISSUE THREAD

**Issue by niklas-joh** [niklas-joh/Machine-Learning-Project-Supervised-Learning#1]
To properly evaluate the performance of each model, it's important to create a training and predicting pipeline that quickly and effectively train models using various sizes of training data and perform predictions on the testing data. The implementation here will be used in all following sections of the project.

Implement the following:

- Import `fbeta_score` and `accuracy_score` from `sklearn.metrics`.
- Fit the learner to the sampled training data and record the training time.
- Perform predictions on the test data `X_test`, and also on the first 300 training points `X_train[:300]`.
- Record the total prediction time.
- Calculate the accuracy score for both the training subset and testing set.
- Calculate the F-score for both the training subset and testing set.
- Make sure to set the beta parameter!
